pkg/utils/args/args_test.go

```
@@ -1,18 +1,25 @@
 package args
 
 import (
+	"net"
 	"testing"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/ginkgo/extensions/table"
 	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/types"
 )
 
 func TestParseArguments(t *testing.T) {
 	RegisterFailHandler(Fail)
 	RunSpecs(t, "ParseArguments Suite")
 }
 
+var parseCIDR = func(s string) net.IPNet {
+	_, cidr, _ := net.ParseCIDR(s)
```

Would asset that the error is nil, to prevent silent failures.